.github/workflows/pr.yml

```diff
@@ -129,7 +114,9 @@ jobs:
             -e COMPILER_BASE_PATH=/compilers \
             -e WINEPREFIX=/tmp/wine \
             -e LOCAL_FILE_DIR=/local_files \
-            decompme_backend -c 'cd /decomp.me && python backend/manage.py test backend'
+            decompme_backend -c 'cd /decomp.me/backend && \
+            /home/user/.local/bin/poetry install && \
```

Comment:
Yeah I'm confused because it should work given that we install them in the Dockerfile, but it doesn't...so I guess we should remove them from the Dockerfile? But that's annoying. I was hoping you could help me figure out why it doesn't work to just execute poetry (why it can't find the deps)